charts/fluentd/tests/fluentd_hpa_test.yaml

```diff
@@ -1,24 +0,0 @@
-suite: Unit tests for Fluentd HPA
```

Comment:
Why is this file removed now? Shouldn't we update it with the memory specs instead?

Reply:
Same question about all the test files that are removed in this PR.

Reply:
The unittest library doesn't seem to support the usage of `.Capabilities.APIVersions.Has` 🤔